_data/express.yml

```diff
@@ -1,2 +1,2 @@
-current_version: "4.19.2"
-next_version: "5.0.0-beta.1"
\ No newline at end of file
+current_version: "4.20.0"
```

Comment:
We should promote version 5 instead of version 4
```suggestion
current_version: "5.0.0"
```

Reply:
We could, but our current version is not 5 yet. We are still calling it next until we polish our documentation or other things first. 

In short, `4.20.0` will be _lastest_ and `5.0.0` will be _next_ for a while.